HW0/P5/P5.py

```diff
@@ -0,0 +1,17 @@
+import matplotlib.pyplot as plt
+import math
+import numpy as np
+
+x = np.linspace(0.5,300,1000);
+
+y1 = [math.log(i,2) for i in x] 
+y2 = [i-1 for i in x] 
+
+p1, = plt.plot(x,y1,'b');
+p2, = plt.plot(x,y2,'r');
+plt.ylim([0,10]);
```

Comment:
You can probably improve the graph by not limiting the Y axis (the idea is to show the general behavior of the methods, and not specific values).